include/mgp.hpp

```diff
@@ -1467,6 +1469,20 @@ inline bool ValuesEqual(mgp_value *value1, mgp_value *value2) {
   if (value1 == value2) {
     return true;
   }
+  // Make int and double comparable, (ex. this is true -> 1.0 == 1)
+  if ((mgp::value_get_type(value1) == MGP_VALUE_TYPE_INT || mgp::value_get_type(value1) == MGP_VALUE_TYPE_DOUBLE) &&
```

Comment:
Can you isolate this check if the value is numeric in some function? So if mgp::value_get_type returns double or int we have numeric, otherwise not